backend/alerts/integrations/webhook/webhook.go

```diff
@@ -99,20 +99,6 @@ func SendUserPropertiesAlert(destination *model.WebhookDestination, payload *int
 	return sendWebhookData(destination, body)
 }
 
-func SendSessionFeedbackAlert(destination *model.WebhookDestination, payload *integrations.SessionFeedbackAlertPayload) error {
-	body, err := json.Marshal(&struct {
-		Event string
-		*integrations.SessionFeedbackAlertPayload
-	}{
-		Event:                       model.AlertType.NEW_USER,
```

Comment:
This was a bug before, it should have been `Event: model.AlertType.SESSION_FEEDBACK`